.github/workflows/test.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         include:
           - ruby: "3.0.2"
             bundler: "2.3.5"
+          - ruby: "3.1"
```

Comment:
Do we want these pegged to a minor version as well? I'm fine being flexible and running against the latest minor version as I doubt there would be a breaking change, but I did want to make sure we are ok with that.